Source/Categories/ESTabBarController+Autolayout.m

```diff
@@ -36,10 +37,16 @@ - (void)setupButtonsConstraints {
         [self.view addConstraints:[self verticalLayoutConstraintsForButtonAtIndex:i]];
         [self.view addConstraint:[self widthLayoutConstraintForButtonAtIndex:i]];
         [self.view addConstraint:[self heightLayoutConstraintForButtonAtIndex:i]];
+        
+        [self.badgeLabels[i] setTranslatesAutoresizingMaskIntoConstraints:NO];
```

Comment:
This method, `setupButtonsConstraints`, is intended to only setup constraints for buttons. Would you rather add a method like `- (void)setupBadgeLabelsConstraints`?

I think it's going to look neater.